oneclick.py

```diff
@@ -328,9 +332,14 @@ def get_metadata_by_isbn(self, identifier):
             raise IOError("OneClick isbn search response not parseable - has no respdict.")
 
         if "message" in respdict:
-            # can happen if searched for item that's not in library's catalog
-            raise ValueError("get_metadata_by_isbn(%s) in library #%s catalog ran into problems: %s" % 
-                (identifier_string, str(self.library_id), respdict['message']))
+            message = respdict['message']
+            if message.startswith("Invalid 'MediaType', 'TitleId' or 'ISBN' token value supplied: "):
+                # we searched for item that's not in library's catalog -- a mistake, but not an exception
+                return None
+            else:
+                # something more serious went wrong
+                raise ValueError("get_metadata_by_isbn(%s) in library #%s catalog ran into problems: %s" % 
```

Comment:
changed.  the unit test for the "item not found" exception is there.  what's the other case you're thinking of?